tests/model/test_experiment_list.py

```diff
@@ -908,35 +908,6 @@ def test_experimentlist_from_file(monkeypatch, dials_regression, tmpdir):
     assert exp_list[0].beam
 
 
-def test_experimentlist_imagesequence_stills():
-    filenames = [
-        "/Users/rjgildea/tmp/118/Puck3_10_1_000%i.cbf.gz" % i for i in range(1, 4)
```

Comment:
Totally agree that this test is useless as stands.

If this test is important, can we either:
 - move _this_ data into `dials_data` - 3 images, reasonable
 - refactor to use existing data from `dials_data`

Reply:
:arrow_right: #353